.travis.yml

```diff
@@ -2,14 +2,10 @@ language: ruby
 sudo: false
 script: rake
 rvm:
-  - 2.2.0
-  - 2.1.5
-  - 2.0.0
-  - 1.9.3
-  - jruby-1.7.9-d19 # fails after 1.7.10 for some reason
-#  - jruby-19mode # JRuby in 1.9 mode
+  - 2.2.3
```

Comment:
Is 2.2.3 also going to be on harvestdor-xxx boxes?  We need to cover the spotlight boxes and the harvestdor-xxx boxes, but that should be it, I think?  

@peetucket  Does Revs index from harvestdor-xxx boxes?  Does it use harvestdor-indexer?